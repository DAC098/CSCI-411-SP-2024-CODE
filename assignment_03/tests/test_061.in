6 10
65 392 431 470 762 970
568
9204
8939
8855
5089
9726
6473
2503
2076
1460
