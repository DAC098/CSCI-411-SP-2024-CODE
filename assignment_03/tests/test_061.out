0 0 0 0 0 0 
2 3 0 0 4 5 
1 2 0 1 10 0 
3 0 0 0 5 5 
0 2 1 5 2 0 
1 1 1 2 4 5 
0 5 1 5 1 1 
3 2 0 0 2 0 
0 0 0 0 0 0 
8 0 0 2 0 0 
